feature/experimental/isis/otg_tests/internal/session/session.go

```diff
@@ -118,30 +123,43 @@ func addISISOC(dev *oc.Root, areaAddress, sysID, ifaceName string) {
 }
 
 // addISISTopo configures basic IS-IS on an ATETopology interface.
-func addISISTopo(iface *ondatra.Interface, areaAddress, sysID string) {
-	isis := iface.ISIS()
-	isis.
-		WithAreaID(areaAddress).
-		WithTERouterID(sysID).
-		WithNetworkTypePointToPoint().
-		WithWideMetricEnabled(true).
-		WithLevelL2()
+func addISISTopo(dev gosnappi.Device, areaAddress, sysID string) {
+
+	devIsis := dev.Isis().
+		SetSystemId(sysID).
+		SetName("devIsis")
+
+	devIsis.Basic().
+		SetHostname(devIsis.Name()).SetLearnedLspFilter(true)
+
+	devIsis.Advanced().
+		SetAreaAddresses([]string{strings.Replace(areaAddress, ".", "", -1)})
+
+	devIsisInt := devIsis.Interfaces().
+		Add().
+		SetEthName(dev.Ethernets().Items()[0].Name()).
+		SetName("devIsisInt").
+		SetNetworkType(gosnappi.IsisInterfaceNetworkType.POINT_TO_POINT).
+		SetLevelType(gosnappi.IsisInterfaceLevelType.LEVEL_2)
+
+	devIsisInt.Advanced().
+		SetAutoAdjustMtu(true).SetAutoAdjustArea(true).SetAutoAdjustSupportedProtocols(true)
```

Comment:
Not too familiar with IS-iS, are these settings needed for the test? (We do not have the equivalent in the IxNetwork ATE implementation).

Reply:
The first 2 are true by default and the 3rd one may not be needed. I will remove all.